code/modules/mob/living/carbon/carbon.dm

```diff
@@ -77,7 +77,7 @@
 		if(!hurt)
 			return
 
-		if(victim.check_block(src, 0, "[name]", LEAP_ATTACK))
+		if(victim.check_block(src, 0, "[name]", LEAP_ATTACK) || !.) // If parent was FALSE that means it got blocked
```

Comment:
I find this super sus

In throw code `. = TRUE` means "it was caught or blocked" and `. = FALSE` means it wasn't

On top of that shouldn't like, all of this code, cancel if it was "blocked"?

Reply:
If we block the code at the top the person being thrown doesn't get stunned.
And if it's not blocked it returns hitby.

Reply:
Testing this fix makes all body throws count as blocked as I anticipated